Package.swift

```diff
@@ -28,7 +28,9 @@ var package = Package(
             targets: ["WordPressAPI"]
         )
     ],
-    dependencies: [],
+    dependencies: [
+        .package(url: "https://github.com/apple/swift-docc-plugin", from: "1.0.0"),
+    ],
```

Comment:
I wish SPM had "development dependencies". There is no need to download this dependency when used in the app.